### ISSUE TITLE
Output path for each agent ?

### ISSUE THREAD

**Issue by miyunluo** [Jiaoyang-Li/CBSH2#1]
Dear author, is the code able to output the trajectory for each agent given the map and agents' info?

**Comment by Jiaoyang-Li** [comment]
Yes, it is stored in icbs.paths

**Comment by miyunluo** [comment]
Thank you very much for replying! Maybe a silly question, but how to generate this icbs.paths file? 
`./CBSH2 --help` does not seem to have an option to save trajectories.

**Comment by Jiaoyang-Li** [comment]
No, icbs.paths is not a file. "paths" is a variable in Class ICSBSearch that stores the paths for all agents. So you need to change the code in the driver.cpp file to output icbs.paths after icbs.runICBSSearch()
